ui/src/app/data-browser/data-browser.module.ts

```diff
@@ -1,8 +1,7 @@
+import {CommonModule} from '@angular/common';
```

Comment:
Nit: Spacing (I am seeing inconsistent spacing everywhere on imports, but here its within one file. I don't think its within the scope of this PR to make consistent project-wide)

Reply:
Done - for this file at least.